lib/widgets/video_viewer_dialog.dart

```diff
@@ -84,22 +70,17 @@ class _VideoViewerDialogState extends State<VideoViewerDialog> {
           )
         ],
       ),
-      child: videoController != null
-          ? Container(
-              decoration: const BoxDecoration(
-                color: MessageContentStyle.backgroundColorVideo,
-              ),
-              padding: VideoViewerDialogStyle.bottomPaddingVideo,
-              child: SafeArea(
-                child: Video(
-                  pauseUponEnteringBackgroundMode: true,
-                  resumeUponEnteringForegroundMode: true,
-                  controls: MaterialVideoControls,
-                  controller: videoController!,
-                ),
-              ),
-            )
-          : const SizedBox(),
+      child: Container(
+        decoration: const BoxDecoration(
+          color: MessageContentStyle.backgroundColorVideo,
+        ),
+        child: Video(
```

Comment:
removed safe area ?
please demo on iphone pro max

Reply:
please demo on media chat details page also

Reply:
okay, you can see it in the link I post above

Reply:
https://github.com/linagora/twake-on-matrix/assets/43041967/6da4a66e-7326-4993-9a2b-61e34868fcf5

Reply:
@sherlockvn  can't see fullscreen player in media page